strongbox-db-liquibase/src/main/resources/db/changelog/v1.0.0/v1.0.0.19__ArtifactEntry.xml

```diff
@@ -0,0 +1,14 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:o="http://www.unbroken-dome.org/schema/liquibase-orientdb"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.2.xsd">
+
+    <changeSet id="v1.0.0.19" author="gryniuk.oleksandr@gmail.com">
+
+        <o:createProperty name="checkSums" type="embeddedmap" linkedType="String" className="ArtifactEntry"/>
```

Comment:
This should be `checksums`, not `checkSums`. Also, as `author`, could you please use the system id that we're using -- `strongbox-dev@carlspring.com`? Thanks! :)

Reply:
@carlspring

Reply:
Good evening @carlspring ,
First I changed the variable name the way you asked but then looked at OrientDB schema for this entity (ArtifactEntry) and saw that all the properties in fact follow the naming convention checkSums (not checksums). See the snapshot attached above.

Thus, do not you think it is better to rename it back to checkSums?

Reply:
If this is the case, then you've spotted something that needs to be corrected. Could you please changes those to `checksums` as well? Thanks!

Reply:
@ogryniuk I believe you see the database state from the moment when you initially used `checkSums` word instead of `checksums`. If you drop `db` directory from the `strongbox-web-core`, install `strongbox-db` with `checksums`, and then restart the `strongbox` app I believe the database will be setup using proper field naming.

Reply:
> If this is the case, then you've spotted something that needs to be corrected. Could you please changes those to checksums as well?

@carlspring I would need a clarification to be sure I will do what you wish. So, you would like that I remove capitalisation from all the properties in ArtifactEntry xml? 

For each of the following properties: storageId, repositoryId, artifactCoordinates, tagSet. artifactPath, sizeInBytes, downloadCount, lastUpdatedlastUsed?

Is it so? Please, confirm...

Reply:
@ogryniuk ,

We are not talking about the decapitalization of everything. Checksum is one word, therefore it should not be `checkSums`. Please, follow @fuss86 advice! Thanks! :)

Reply:
@carlspring 
In such case, know that it is already renamed checksums, as you can see it in the updated snapshot

Reply:
Okay, thanks for changing it! :)